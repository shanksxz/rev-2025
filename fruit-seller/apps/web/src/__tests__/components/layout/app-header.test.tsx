import { render } from '@testing-library/react';
import AppHeader from '@/components/layout/app-header';
import CartContextProvider from '@/hooks/use-cart';

describe('AppHeader', () => {
    it('should match snapshot', () => {
        const { container } = render(
            <CartContextProvider>
                <AppHeader />
            </CartContextProvider>
        );
        expect(container).toMatchSnapshot();
    });
});
