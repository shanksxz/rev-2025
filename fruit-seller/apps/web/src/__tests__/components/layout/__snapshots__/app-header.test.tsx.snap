// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader should match snapshot 1`] = `
<div>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hqwy3n-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-u5lowu-MuiToolbar-root"
    >
      <h6
        class="MuiTypography-root MuiTypography-h6 css-19xrsc3-MuiTypography-root"
      >
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-7gp6pd-MuiTypography-root-MuiLink-root"
          href="/"
        >
          Ecomm
        </a>
      </h6>
      <div
        class="MuiBox-root css-huskxe"
      >
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-19qjfsi-MuiTypography-root-MuiLink-root"
          href="/products"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-1mdftu-MuiTypography-root"
          >
            Explore
          </p>
        </a>
        <a
          class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-19qjfsi-MuiTypography-root-MuiLink-root"
          href="/cart"
        >
          <span
            class="MuiBadge-root css-1c32n2y-MuiBadge-root"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-qrdpap-MuiSvgIcon-root"
              data-testid="ShoppingCartIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2M1 2v2h2l3.6 7.59-1.35 2.45c-.16.28-.25.61-.25.96 0 1.1.9 2 2 2h12v-2H7.42c-.14 0-.25-.11-.25-.25l.03-.12.9-1.63h7.45c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.08-.14.12-.31.12-.48 0-.55-.45-1-1-1H5.21l-.94-2zm16 16c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2"
              />
            </svg>
            <span
              class="MuiBadge-badge MuiBadge-standard MuiBadge-invisible MuiBadge-anchorOriginTopRight MuiBadge-anchorOriginTopRightRectangular MuiBadge-overlapRectangular MuiBadge-colorPrimary css-y2h6q0-MuiBadge-badge"
            />
          </span>
        </a>
      </div>
    </div>
  </header>
</div>
`;
